README.md

```diff
@@ -106,6 +106,9 @@ TAG Contributor Strategy has working groups and teams to effectively collaborate
 
 Status: [Proposed](https://github.com/cncf/tag-contributor-strategy/issues/365)
 
+### Contributor Strategy Overview Diagram
+<img src="website/static/images/TAG-CS_roadmap_visual.png" alt="an image describing TAG Contributor strategy roadmap" align="middle" width="800px">
```

Comment:
```suggestion
<img src="website/static/images/TAG-CS_roadmap_visual.png" alt="an image describing TAG Contributor strategy roadmap">
```

`align` is obsolete, so removing it.

Leaving the `width` makes things better. Images on markdown are by default 100% wide anyway.

Reply:
Happy to merge after this change is addressed